ayekan/exporters/snmp-generator/Makefile

```diff
@@ -0,0 +1,356 @@
+# Copyright 2018 The Prometheus Authors
```

Comment:
Maybe add a comment that this file was copied from https://raw.githubusercontent.com/prometheus/snmp_exporter/main/generator/Makefile ?